src/gateio_new_coins_announcements_bot/announcement_scrapers/binance_scraper.py

```diff
@@ -18,7 +20,17 @@ def fetch_latest_announcement(self):
         """
         logger.debug("Pulling announcement page")
         request_url = self.__request_url()
-        response = self.http_client.get(request_url)
+
+        if rotating_proxy_is_ready():
+            proxy = get_proxy()
+            logger.debug(f"Using proxy: {proxy}")
```

Comment:
I think this should be a logger.info combined with the first message e.g.:
when using proxy: "Getting Binance announcements [Using proxy: 127.0.0.1]"
when no valid proxy exists: "Getting Binance announcements [No proxy available]"
when the proxy feature is disabled: "Getting Binance announcements"

That being said, it should be implemented in a new PR and not here.
I was planning on reworking the console logs at some point anyway so we should consider this then as well.